### PR TITLE
ss-184: fix visual defects on landing page

### PR DESCRIPTION
**Pull request by Evcat6**

### CODE REVIEW

File: frontend/src/bundles/landing/components/button-tabs/button-tab.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+};
+
+const ButtonTab: React.FC<Properties> = ({ title, isActive, disabled }) => {
+    const tabClassName = classNames(styles.tab, {
+        [styles.active]: isActive,
+        [styles.disabled]: disabled,
+    });
+    return (
+        <button disabled={disabled} className={tabClassName}>
+            {title}
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/landing/components/button-tabs/button-tab.tsx:18]: I think it should be 'label'

File: frontend/src/bundles/landing/components/button-tabs/button-tab.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+type Properties = {
+    title: string;
+    isActive: boolean;
+    disabled: boolean;
+};
+
+const ButtonTab: React.FC<Properties> = ({ title, isActive, disabled }) => {
+    const tabClassName = classNames(styles.tab, {
+        [styles.active]: isActive,
+        [styles.disabled]: disabled,
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/landing/components/button-tabs/button-tab.tsx:14]: You can use .button:disabled in  styles

File: frontend/src/bundles/landing/components/button-tabs/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+.tab:hover {
+    cursor: default;
+}
+
+.tabs:last-child {
+    margin-right: 0;
+}
+
+.tab.active {
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/landing/components/button-tabs/styles.module.scss:36]:
```suggestion
.activeTab {
```

File: frontend/src/bundles/landing/components/feedbacks-part/feedbacks-part.tsx [hunk 4]
```diff
@@ -10,20 +17,53 @@ type Properties = {
...
+    horizontalClass: horizontalClass,
+    clickable: true,
+};
+
+const breakpoints = {
+    992: {
+        slidesPerView: 3,
+        spaceBetween: 35,
+    },
+    768: {
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/landing/components/feedbacks-part/feedbacks-part.tsx:36]: Add an enum with these values

File: frontend/src/bundles/landing/components/feedbacks-part/styles.module.scss [hunk 5]
```diff
@@ -1,77 +1,88 @@
...
 
 .title {
+    margin-top: 0;
     color: var(--color-navy-900);
     font-size: var(--font-size-level-1-heading);
     line-height: var(--line-height-level-1-heading);
 }
 
-@media (max-width: 768px) {
+@media (max-width: 1200px) {
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/landing/components/feedbacks-part/styles.module.scss:57]: Replace with variable

File: frontend/src/bundles/landing/components/third-step-part/styles.module.scss [hunk 6]
```diff
@@ -1,95 +1,104 @@
...
+        bottom: -20px;
     }
 
-    .chartCardsContainer {
-        position: inherit;
-        display: flex;
-        flex-direction: column;
-        gap: 20px;
-    }
+    // .container {
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/landing/components/third-step-part/styles.module.scss:46]: Remove commented code

Comment (Evcat6) [reply frontend/src/bundles/landing/components/third-step-part/styles.module.scss]: Resoved in [214d526](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/214d526cd24228787ecabce6767c0fe9ecec563f)

File: frontend/src/bundles/landing/components/feedbacks-part/styles.module.scss (hunk 5 above)
Comment (Evcat6) [reply frontend/src/bundles/landing/components/feedbacks-part/styles.module.scss:57]: Resolved in [d153e37](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/d153e379cd5c556a068d7900f44e914546b8ee72)

File: frontend/src/bundles/landing/components/feedbacks-part/feedbacks-part.tsx (hunk 4 above)
Comment (Evcat6) [reply frontend/src/bundles/landing/components/feedbacks-part/feedbacks-part.tsx]: Resolved in [0b210a5](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/0b210a5e85aba7993ed3dbd680ec2d4640f94eab)

File: frontend/src/bundles/landing/components/button-tabs/button-tab.tsx (hunk 1 above)
Comment (VItaliy-Kalenichenko) [reply frontend/src/bundles/landing/components/button-tabs/button-tab.tsx:17]:
```suggestion
            {label}
```

Comment (Evcat6) [reply frontend/src/bundles/landing/components/button-tabs/button-tab.tsx]: Resolved in [52f5038](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/52f5038f8266bba4cd7f2479a2d33a26612d7116)

File: frontend/src/bundles/landing/components/button-tabs/styles.module.scss (hunk 3 above)
Comment (Evcat6) [reply frontend/src/bundles/landing/components/button-tabs/styles.module.scss]: Resolved in [834a319](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/834a319e247b1efe74173fcd33bc96a78504a5aa)

**Pull request merged**